core/domain/prod_validation_jobs_one_off.py

```diff
@@ -6183,6 +6184,83 @@ def _get_external_instance_custom_validation_functions(cls):
         return [cls._validate_reference]
 
 
+class PlatformParameterModelValidator(BaseModelValidator):
+    """Class for validating PlatformParameterModel."""
+
+    @classmethod
+    def _get_model_id_regex(cls, unused_item):
+        return '^.*$'
```

Comment:
This is too lenient, why not the default id regex, why are you overriding?

Reply:
The default regex only allows the id length to be up to 12, which is not sufficient for feature names. But you are right that it's too lenient.
Changed them to allow only allow alphanumeric characters and underscores.